examples/example-app/src/main/scala/org/virtuslab/example/StatsService.scala

```diff
@@ -0,0 +1,87 @@
+package org.virtuslab.example
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+
+import io.circe.Codec
+import io.circe.generic.semiauto.deriveCodec
+
+import scala.concurrent.duration._
+
+import org.virtuslab.ash.circe.AkkaCodecs
+
+//#service
```

Comment:
Nit: these `//#...` comments can probably be removed, it seems they're only needed for some Akka docs generation (?)